user/migrate/travis-migrate-to-apps-gem-guide.md

```diff
@@ -1,18 +1,20 @@
 ---
-title: Migrating Multiple Repositories to GitHub Apps Guide
+title: Migrating 50 or more private Repositories to GitHub Apps Guide
 layout: en
 ---
 
-If your organization has more than **50 active repositories** to migrate to our new GitHub Apps integration, you can use [the travis_migrate_to_apps gem](https://github.com/travis-ci/travis_migrate_to_apps) to migrate them in one sweep rather than having to add them manually one by one in the GitHub UI.
+If your organization has more than **50 active repositories** to migrate to our new GitHub Apps integration, you can use [the travis_migrate_to_apps gem](https://github.com/travis-ci/travis_migrate_to_apps) to migrate them in one sweep rather than adding them manually one by one in the GitHub UI.
 
-Here are the steps:
+> Note that you can also choose "Activate all repositories" on GitHub to select all current and future repositories, so that you can activate them at Travis CI by adding a `.travis.yml` file.
```

Comment:
I think this is perfectly fine, just being picky since the combo "activating" + "adding a .travis.yml file" (still) sounds a bit strange to me. What do you think about:
> "so that your projects start building as soon as they're configured with a `.travis.yml` file."

? 🤔 
Is this too verbose/too much? hahaha

Reply:
that's perfect, thanks!